transform/mattermost-analytics/models/marts/release/dim_fix_versions.sql

```diff
@@ -0,0 +1,36 @@
+
+WITH unparsed_fix_versions AS (
```

Comment:
should we also handle smth. like `triage` in the fix versions names set?

Reply:
Following discussion with @amyblais, only fix versions with a version in them (vX.Y) have some value for the current use case. Additional conventions in "fix version" field can be added in the future, if needed.